src/pages/Inventory.page.js

```diff
@@ -9,6 +9,22 @@ export class InventoryPage extends BaseSwagLabPage {
 
     addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
 
+    sortField = this.page.locator('[data-test="product-sort-container"]');
+
+    inventoryItemsName = this.page.locator('[data-test="inventory-item-name"]');
+
+    inventoryItemsPrice = this.page.locator('[data-test="inventory-item-price"]');
+
+    goToCart = this.page.locator('[data-test="shopping-cart-link"]');
+
+    inventoryItemsNameRandom(index) {return this.page.locator(`(//div[@data-test="inventory-item-name"])[${index}]`)};
```

Comment:
Why do we have random in the method name? Please go though your methods (all) and recheck naming. [Naming convs](https://gazar.dev/clean-code/best-practice-naming-conventions-typescript) here are not full, but they should help a bit. Also I recommend[ this topic](https://uk.javascript.info/ninja-code).